src/libcrun/criu.c

```diff
@@ -645,8 +649,11 @@ libcrun_container_restore_linux_criu (libcrun_container_status_t *status, libcru
       goto out_umount;
     }
 
-  /* If there is a PID or network namespace defined in config.json we are telling
-   * CRIU to restore the process into that namespace.
+  bool join_ns_support = criu_check_version (31601) == 1;
```

Comment:
Could we add this under macro `#ifdef CRIU_JOIN_NS_SUPPORT` so clang knows when is it being used

Reply:
Thank you @flouthoc, I've added `#ifdef` around `join_ns_support` but kept it outside the for loop.